sys/include/net/conn.h

```diff
@@ -15,6 +15,7 @@
  * About
  * =====
  *
+ * @code
```

Comment:
Oh ok sure, I used pktbuf.h as a reference (which uses `@code`as well), should I change that too while I'm at it?